lib/saluki-io/src/net/client/http.rs

```diff
@@ -64,6 +64,7 @@ where
         HttpClient {
             inner: Client::builder(TokioExecutor::new())
                 .pool_max_idle_per_host(5)
+                .pool_idle_timeout(Duration::from_secs(45))
```

Comment:
Looks like we're missing setting the pool timer as the docs [mention](https://docs.rs/hyper-util/latest/hyper_util/client/legacy/struct.Builder.html#method.pool_idle_timeout) needing to do.